lib/open_food_network/cached_products_renderer.rb

```diff
@@ -27,19 +27,21 @@ def products_json
     private
 
     def cached_products_json
-      if Rails.env.production? || Rails.env.staging?
-        Rails.cache.fetch("products-json-#{@distributor.id}-#{@order_cycle.id}") do
-          begin
-            uncached_products_json
-          rescue ProductsRenderer::NoProducts
-            nil
-          end
+      return uncached_products_json unless use_cached_products?
+
+      Rails.cache.fetch("products-json-#{@distributor.id}-#{@order_cycle.id}") do
+        begin
+          uncached_products_json
+        rescue ProductsRenderer::NoProducts
+          nil
         end
-      else
-        uncached_products_json
       end
     end
 
+    def use_cached_products?
+      Spree::Config[:enable_products_cache?] && (Rails.env.production? || Rails.env.staging?)
```

Comment:
What if we just set `enable_products_cache?` from each of the different `config/environments/` files: development, production, staging and test? Then we wouldn't need the `&&` at all and we could handle this config without touching this code :ok_hand:

Reply:
I disagree because super admin would not be able to run the following test:

Reply:
I'm was suggesting a different way to implement what you did in this line. No change in behavior at all, but now that I read the line again I don't understand why we exactly need the `(Rails.env.production? || Rails.env.staging?)` part of the conditional.

Reply:
yes, cache is disabled in dev by default. that's it.